### ISSUE TITLE
Task page shows completion that was never created

### ISSUE THREAD

**Issue by andreaferretti** [heartexlabs/label-studio#427]
**Describe the bug**
Every new task shows some kind of completion, even before starting labeling anything.

**To Reproduce**
1. Import some tasks
2. Open any task
3. Under completions, see `New completion Created undefined by Awesome User`

**Expected behavior**
The completion list should be empty

**Screenshots**
<img width="1412" alt="completion" src="https://user-images.githubusercontent.com/1962818/93459950-d2229300-f8e2-11ea-9d66-10ceb5fdf894.png">

**Comment by andreaferretti** [comment]
I understand. It is still strange to see that, though, for various reasons:

* who is Awesome user?
* the completion looks like any real completion (for instance, it can be deleted, it can be set as ground truth)
* it is not clear that the completion is empty (my first istinct was to try and figure out what was in there)

I suggest to not show this temporary completion on the frontend side, even if it is necessary to create it for technical reasons

**Comment by makseq** [comment]
1. > who is Awesome user

I've removed Awesome user, and now it's a little bit clearly: 
![image](https://user-images.githubusercontent.com/501780/93668376-8b749a80-fa94-11ea-8208-e0ccf25dbb63.png)

2 & 3.
> the completion looks like any real completion
> it is not clear that the completion is empty
But it has a blue point at the left and name "New completion".
Yes, it's a little bit confusing, I think we will rework this mechanics in future.

**Comment by andreaferretti** [comment]
Thank you, this is already much more clear
